README.md

```diff
@@ -345,6 +342,25 @@ analysis information from the hub.
       ```
       socketTimeoutMS: 300000,
       ```
+    * To specify a CodeSonar analysis ID of `<X>` which the CodeSonar plugin should use for evaluating analysis results, replace the `projectFile` line with the following.
+      ```
+      aid: <X>,
+      ```
+      For example, to specify an analysis ID of 81:
+      ```
+      aid: 81,
+      ```
+    * To specify a custom warning filter <X> for identifying new warnings, edit the last lines of the `codesonar` plugin invocation as follows.
```

Comment:
similarly, change \<X\> to something meaningful/unique (within size constraints), e.g. \<NW\>